app/views/admin/users/widgets/_organizations.html.erb

```diff
@@ -25,21 +25,23 @@
         </h3>
       </div>
       <div class="c-card__revealable shrink-0">
-        <button type="button" class="c-btn c-btn--icon-alone" data-modal-trigger data-modal-title="Edit organization" data-modal-size="small" data-modal-content-selector="#edit-organization-<%= org_membership.organization.id %>">
+        <button type="button" class="c-btn c-btn--icon-alone" data-modal-trigger data-modal-title="Edit organization" data-modal-size="small" data-modal-content-selector="#edit-organization-<%= org_membership.organization.id %>" aria-label="Edit <%= org_membership.organization.name %> organization membership">
```

Comment:
@JennieOcken @Ridhwana, are you both OK with this wording?

Reply:
Cc @amywtlin who I think is working on the copy for the project.

Reply:
Thank you, @Ridhwana! @nickytonline Can you let me know what is the best way to see where this appears on the interface?

Reply:
You would have to run Forem locally or visit one of our forem like the Ben Halpern community or Canary and enable the feature flag `admin-member-view`.

Reply:
Can you enable the feature flag on Canary 1 so @amywtlin can take a look at it in UI when she is back on?  We don't have the access to turn on and off feature flags (and don't want it at this time).